Buckley will initiate conversation of profile created

Js-o opened the pull request:
1. Added post hook that will pass in created profile to function intro
2. Will ask two consecutive question, each time updating name and location respectively

Commit:
Empty folder for bot store

to transfer to database

Commit:
Commit before pull rebase from team/user/profile for onboard flow

rebasing to Jeff's working copy

Commit:
Commit before pull rebase from team/user/profile for onboard flow

rebasing to Jeff's working copy

Commit:
Set up intro for buckley bot on profile creation

Should initiate conversation on profile creation

Commit:
Buckley will initiate conversation of profile created

1. Added post hook that will pass in created profile to function intro
2. Will ask two consecutive question, each time updating name and location respectively

Commit:
Removing askName and askLocation from bot.js

Moved into the bot/introduction.js for Buckley inroduction

Commit:
Changing findAll to find in line 7

Changed user[0] to user due to find instead of findAll

pranaymohan commented on server/src/server.js:
```diff
@@ -1,6 +1,6 @@
 import express from 'express';
 import bodyParser from 'body-parser';
-import bot from './bot';
+import bot from './bot.js';
```
don't think you need .js extension? (We should be including it in our webpack resolve extensions)

pranaymohan commented on server/src/bot.js:
```diff
@@ -2,32 +2,23 @@ import Botkit from 'botkit';
 import dotenv from 'dotenv';
```
Do we need this anymore? I think .env works without it?? Not sure, pls confirm w/ rest of team

Js-o replied:
don't need the .env file to work. Example seen in teamController

Will remove on next pr

Commit:
Completed bot to initiate conversation on new team

1. Refactored to include support of multiple teams and instance of bots, within bot.js store with the format {teamid: instance of bot}
2. Added a hook post team create in teamModel.js to re-generate the object store stated in 1.
3. Need to further research what happens when you do not reply the conversation in slack
